docs/cli-arguments.md

```diff
@@ -25,35 +25,33 @@ spec:
 ```txt
 $ kube-state-metrics -h
 Usage of ./kube-state-metrics:
-      --add_dir_header                              If true, adds the file directory to the header
```

Comment:
Besides from here, we also need to remove them from the docs. https://github.com/kubernetes/kube-state-metrics/blob/f09db29f7955e5e628c54130fac1ac116836412c/docs/node-metrics.md

That should fix the CI.